.github/workflows/deployment_tests.yml

```diff
@@ -232,19 +222,19 @@ jobs:
 #          terraform apply plan.tfplan
 #
 #      - name: Destroy GKE clusters
-#        if: steps.deploy-gke.outcome != 'skipped'
+#        if: always()
 #        run: |
 #          cd gke
 #          terraform destroy -auto-approve
 
       - name: Destroy Google Service Accounts
-        if: steps.deploy-GSAs.outcome != 'skipped'
+        if: always()
```

Comment:
Why do we need an `if` it is `always`?

Reply:
Interesting. Like an exception handler. Worth recording on any documentation we may have for BB Pipelines.